reactioncommerce/api-plugin-simple-schema
# feat: move files from reaction to plugin

This plugin replaces the internal `simple-schema` plugin. 

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- see that the plugin is loaded upon startup in Reaction (use the `systemInformation` query)
- See that any `simple-schema` is loaded as expected. You can do this by doing almost any mutation in reaction, which should check a schema.
